setup.py

```diff
@@ -19,9 +19,11 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub>=1.25.2',
```

Comment:
@cchristous you will be happy to know that we are now installing PyGithub from PyPi. They released yesterday and it included my changes. Now more "dependency_links"